lenovo/loq/15aph8/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, ...}: {
+  imports = [
+    ../../../common/cpu/amd/pstate.nix
+    ../../../common/gpu/nvidia/prime.nix
+    ../../../common/gpu/nvidia/ada-lovelace
+    ../../../common/pc/laptop
+    ../../../common/pc/laptop/ssd
+  ];
+
+  hardware.nvidia.prime = {
+    amdgpuBusId = lib.mkDefault "PCI:5:0:0";
+    nvidiaBusId = lib.mkDefault "PCI:1:0:0";
+  };
+
+  # Power management - using mkDefault as these can be overridden
+  hardware.nvidia.powerManagement.enable = lib.mkDefault true;
+  hardware.nvidia.modesetting.enable = lib.mkDefault true;
```

Comment:
IIUC this line is implied by the call to `gpu/nvidia/ada-lovelace`, as it calls `gpu/nvidia`, containing this same line.
(Notably the mentioned PR there is already merged)